### PR TITLE
Content header refactor

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: assets/sass/patterns/organisms/content-header.scss [hunk 1]
```diff
@@ -40,9 +40,9 @@
   @include margin($content-header-padding, "bottom");
 
   .content-header--header & {
-    @include margin($content-header-padding * 1.5 + 11 * 2.1818181818, "top"); // 11 * 2.1818181818 is the height of the subject line
+    @include margin($content-header-padding / 2, "top"); // 11 * 2.1818181818 is the height of the subject line
```
Comment (nlisgo) [inline assets/sass/patterns/organisms/content-header.scss:43]: We should remove or amend the note after this line as it not longer applies.

File: assets/sass/patterns/organisms/content-header.scss [hunk 2]
```diff
@@ -401,8 +398,8 @@ $iconlist: cc oa;
   background-repeat: no-repeat;
   background-position: center bottom;
   display: block;
-  width: 17px;
   height: 22px;
+  width: 17px;
```
Comment (nlisgo) [inline assets/sass/patterns/organisms/content-header.scss:402]: this can be undone.

File: assets/sass/patterns/organisms/content-header.scss [hunk 3]
```diff
@@ -292,25 +292,24 @@
 
 }
 
+.content-header__component-assets {
+  display: grid;
+  grid-template-rows: auto auto auto;
```
Comment (nlisgo) [inline assets/sass/patterns/organisms/content-header.scss:297]: I believe that this could be shortened to: `grid-template-rows: auto;`

**Review by nlisgo — APPROVED** [review]
Nothing blocking, just a few comments that you may want to pick up on.

**Comment by nlisgo** [comment]
I suggest it would be valuable to add or amend a variant of the content header to cause this issue within pattern-library.

File: assets/sass/patterns/organisms/content-header.scss (hunk 3 above)
Comment (discodavey) [reply assets/sass/patterns/organisms/content-header.scss:297]: I am changing this.

**Comment by NuclearRedeye** [comment]
It's subtle, but they're aren't quite in the same positions as before. It's neither here nor there for me, but you should get product to sign off on the changes..>

On production...
Screenshot 2020-11-16 at 08 29 49

Preview of your changes...
Screenshot 2020-11-16 at 08 30 30

**Comment by discodavey** [comment]
@nlisgo Can we get this onto continuumtest please so we can check this properly?

**Pull request merged**